test/extended/apiserver/kubeconfigs.go

```diff
@@ -80,3 +104,28 @@ func testNode(oc *exutil.CLI, kubeconfig, masterName string) error {
 	}
 	return nil
 }
+
+func testKubeApiserverContainer(oc *exutil.CLI, kubeconfig, masterName string) error {
+	g.By("Testing kube-apiserver container on master node " + masterName)
+	kubeconfigPath, ok := kubeApiserverLocations[kubeconfig]
+	if !ok {
+		return fmt.Errorf("location for %s kubeconfig not found", kubeconfig)
+	}
+
+	framework.Logf("Copying oc binary from host to kube-apiserver container in master %q", masterName)
+	out, err := oc.AsAdmin().Run("debug").Args("node/"+masterName, "--", "chroot", "/host", "/bin/bash", "-euxo", "pipefail", "-c",
```

Comment:
refining these calls to use the standard kubeClient and avoiding the process fork to `oc` would be a significant improvement for reliability (code controlled retries) and debugging (we get better error messages).  Willing to accept as a followup.